src/index.html

```diff
@@ -13,6 +13,54 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <body>
+      <div class="stars stars--0">
```

Comment:
Use semantic tags where possible. Here, a <section> or <article> tag could be more appropriate to encapsulate the stars ratings.